tests/framework/artifact/test_manager.py

```diff
@@ -57,7 +59,12 @@ def test_subset_rows():
 def test_subset_columns():
     values = [0, "red", 100]
     data = build_table(
-        values, 1990, 2010, columns=("age", "year", "sex", "draw", "color", "value")
+        values,
+        parameter_columns={
```

Comment:
Why did you need to specify the parameter columns here?

Reply:
To change the range of the year column to keep it the same as it was since it is different from the default - 2010 vs 2020

Reply:
I understand that. I'm saying that the specific endpoints probably don't matter for this test, so you should be able to use the defaults.